WalletWasabi/Legal/Assets/LegalDocumentsWw2.txt

```diff
@@ -78,7 +78,7 @@ I. TERMS AND CONDITIONS
      -The UTXO comes from a One Hop transaction.
```

Comment:
I guess here we have to specify that we are talking about Wasabi coinjoin transactions, otherwise one can use any other service to coinjoin and then tries to remix with Wasabi.

Also we didn't mention the free remixes from a WW2 coinjoin transaction.

Reply:
According to @lontivero's tweet https://twitter.com/lontivero/status/1537141378475597825 there are 3 cases for free coinjoin:

* Coins coming from a WW coinjoin (remixes)
* Coins with value < 0.01btc
* Coins coming with a WW coinjoin ancestor (only one hop)

Why are we saying `The UTXO comes directly from a previous Wasabi Wallet 1.x CoinJoin transaction.`?
Should we not mention the second case in Lucas' tweet?